modules/sdk-core/src/bitgo/wallet/iWallet.ts

```diff
@@ -111,6 +111,7 @@ export interface PrebuildTransactionOptions {
    * This feature is supported only for specific coins, like ADA.
    */
   senderAddress?: string;
+  senderWalletId?: string;
```

Comment:
Please add a jsdoc. As a integrator, I have no idea what this field does and when to supply it.

Reply:
added